### PR TITLE
DCD-1294 security fix quickstart Bitbucket

### PR DESCRIPTION
**Pull request by nghazali**
Security vulnerabilities:

Added proper resources to IAM policies and remove public access on RDS DB instances.

### CODE REVIEW

**Comment by errcode1202** [comment]
Have we performed a backward compatibility test with these changes i.e what happens if a stack created prior to these changes, is then updated with these changes?

**Comment by nghazali** [comment]
I testes the changes with two scenarios, create a new stack, and update the existing stack using the changes in this PR. The changes which required recreate an object is already rolled out and the existing changes are backward compatible.

File: templates/quickstart-bitbucket-dc.template.yaml [hunk 1]
```diff
@@ -727,33 +727,65 @@ Resources:
...
+                  - ec2:DetachVolume
+                  - ec2:AttachVolume
+                  - ec2:CreateTags
+                  - ec2:CreateSnapshot
+                  - ec2:CreateVolume
+                Resource:
+                  - !Sub "arn:${AWS::Partition}:ec2:*:${AWS::AccountId}:instance/*"
+                  - !Sub "arn:${AWS::Partition}:ec2:*:${AWS::AccountId}:volume/*"
+                Effect: Allow
+              - Action:
```
Comment (nanux) [inline templates/quickstart-bitbucket-dc.template.yaml:740]: Can we group the permissions by service?

**Review by nanux — APPROVED** [review]

File: templates/quickstart-bitbucket-dc.template.yaml (hunk 1 above)
Comment (nghazali) [reply templates/quickstart-bitbucket-dc.template.yaml:740]: Done

**Pull request merged**